go.mod

```diff
@@ -4,37 +4,37 @@ go 1.18
 
 require (
 	github.com/aws/aws-sdk-go v1.38.41
-	github.com/backube/volsync v0.5.0
+	github.com/backube/volsync v0.3.1-0.20230117132338-70b044830613
```

Comment:
Should this be going back from 0.5.0 (to 0.3.1...)?

Reply:
@ShyamsundarR it's due to a bit of weirdness with the labels that happen to be on the `main` branch in VolSync we think - essentially I need some api updates in main but they aren't available as a labelled release yet (they will eventually be in 0.7.0).  Essentially I pulled one of the latest commits in main but go gets an old 0.3.0 and updates to the one you see above.... I had discussed with John a bit, and will talk to him again - we might consider labelling a 0.7.0-rcXX so we would be able to pull by that label instead prior to 0.7.0 being released.

Reply:
Thanks for the update Tesshu!

Reply:
Created a tracker for the same #713